tests/suites/cloud/tests/preload/index.js

```diff
@@ -64,6 +64,6 @@ module.exports = {
       return await this.cloud.balena.models.device.isTrackingApplicationRelease(
         this.balena.uuid
       );
-    }, false);
+    }, false, 60, 5 * 1000);
```

Comment:
Same as my previous comment, the shorter interval is good but now the max is 5min vs 10min which may not be enough for very slow devices?